ensime_shared/ensime.py

```diff
@@ -857,7 +861,9 @@ def client_keys(self):
 
     def client_status(self, config_path):
         """Get the client status of a given project."""
-        c = self.client_for(config_path, create=True)
+        # XXX: `create` wasn't used, should this be `create_client` or
+        # `create_classpath`?
+        c = self.client_for(config_path, create_client=True)
```

Comment:
This is a minor change that I did in my last PR. Could you please remove this change?